extensions-contrib/parquet-extensions/README.md

```diff
@@ -0,0 +1,420 @@
+[![Build Status](https://travis-ci.org/druid-io/druid.svg?branch=master)](https://travis-ci.org/druid-io/druid) [![Inspections Status](https://img.shields.io/teamcity/http/teamcity.jetbrains.com/s/OpenSourceProjects_Druid_Inspections.svg?label=TeamCity%20inspections)](https://teamcity.jetbrains.com/viewType.html?buildTypeId=OpenSourceProjects_Druid_Inspections) [![Coverage Status](https://coveralls.io/repos/druid-io/druid/badge.svg?branch=master)](https://coveralls.io/r/druid-io/druid?branch=master)
+
+## Druid - Parquet extension
+
+#### Ingestion using Parquet format
+To use this extension, make sure to include both `druid-avro-extensions` and `druid-parquet-extensions`.
+
+This extension enables Druid to ingest and understand the Apache Parquet data format offline.
+
+#### Parquet Hadoop Parser
+This is for batch ingestion using the HadoopDruidIndexer. The inputFormat of inputSpec in ioConfig must be set to `"io.druid.data.input.parquet.DruidParquetInputFormat"`.
+
+| Field	| Type       | Description           | Required  |
+| ------|:-------    |:-------------:        | :-----|
+| type |String          | This should say `parquet`        | yes |
+| parseSpec |JSON Object          | Specifies the timestamp and dimensions of the data. Should be a timeAndDims parseSpec.              |   yes |
+| binaryAsString |Boolean     |  Specifies if the bytes parquet column should be converted to strings.  |    no (default == false) |
+| parquetParser| JSON Object | Specifies field definitions on how a specific parquet column has to be parsed, Should be a `ParquetParser` spec | no (default == false) |
+
+When the time dimension is a DateType column, a format should not be supplied. When the format is UTF8 (String), either auto or a explicitly defined format is required.
+
+#### Parquet parser
+A sample `parquetParser` spec looks line the one below
```

Comment:
For these examples, can you also include a JSON dump here of `events_with_all_datatypes.parquet`? It would make it easier to understand how field definitions in the example parser specs map to a parquet file.

Reply:
Yeah, that makes sense. We will attach a sample row of the parquet file.